mimecast/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "537f740f-1612-4ced-bddd-1bc666a56ed3",
+  "app_id": "mimecast",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gain insights into Mimecast logs.",
+    "title": "Mimecast",
+    "media": [],
+    "classifier_tags": [
+      "Category::Log Collection",
+      "Category::Security",
+      "Submitted Data Type::Logs"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": false,
+      "source_type_id": 18241205,
+      "source_type_name": "mimecast",
```

Comment:
Please use the natural casing of the name here: "Mimecast"

Reply:
Done. Thanks..!!